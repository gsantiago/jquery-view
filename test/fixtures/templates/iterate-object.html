<ul>
  <li :repeat="user in users">
    {{$index}}: <a href="mailto:{{user}}">{{$key}}</a>
  </li>
</ul>