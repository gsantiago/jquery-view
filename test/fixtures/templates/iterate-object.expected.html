<ul>
  <li>
    0: <a href="mailto:guilherme@email.com">Guilherme</a>
  </li><li>
    1: <a href="mailto:junin@email.com">Junin</a>
  </li><li>
    2: <a href="mailto:zillao@email.com">Zillao</a>
  </li>
</ul>